src/main/scala/org/broadinstitute/dsde/firecloud/model/Library.scala

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.dsde.firecloud.model
```

Comment:
combine `Library.scala` and `Search.scala`? I have no opinion what it should be named, but the model classes feel like they belong together.